QUEST_LV_0100.tsv

```diff
@@ -8094,34 +8094,34 @@ QUEST_LV_0100_20150908_008093
 QUEST_LV_0100_20150908_008094	
 QUEST_LV_0100_20150908_008095	
 QUEST_LV_0100_20150908_008096	
-QUEST_LV_0100_20150918_008097	This place became dangerous because of monsters emerging since Medis Diena.{nl}Quickly return while you can, because Kepa's will soon be marching in.
-QUEST_LV_0100_20150918_008098	You were telling that we need to assemble, right?{nl}Please search my luggage first and return quickly, so I'll go as soon as I can.
-QUEST_LV_0100_20150918_008099	Have you tried offering your worship to a Goddess Statue yet?{nl}The Goddesses might have forsaken us, but I still believe in them.
+QUEST_LV_0100_20150918_008097	$This place has become dangerous ever since monsters emerged on Medis Diena.{nl}Go back quickly while you still can, Kepas will overrun this position soon.
+QUEST_LV_0100_20150918_008098	$Ah, you said you came to tell me about the order to assemble, right?{nl}If you help me to find my belongings, I will be able to do so more quickly.
+QUEST_LV_0100_20150918_008099	$Have you ever tried offering your worship to a Goddess Statue?{nl}The Goddesses might have vanished, but I believe they are still out there somewhere.
 QUEST_LV_0100_20150918_008100	I was worried that the monsters caused harm to the Goddess Statue.{nl}Even though the world is messed up.. It's terrible.
-QUEST_LV_0100_20150918_008101	Hey, I'm really in a bind.{nl}Who knew that the Kepa could grow so big.. Is it because of Medis Diena?
-QUEST_LV_0100_20150918_008102	Don't forget to worship a Goddess Statue when you see one.{nl}It'll give you a blessing. I hope the Goddess hasn't forgotten about us.
+QUEST_LV_0100_20150918_008101	$This has really gotten out of hand.{nl}Who knew that Kepas could grow so big...{nl}I wonder if it's because of Medis Diena?
+QUEST_LV_0100_20150918_008102	Don't forget to worship a Goddess Statue when you see one.{nl}I'm sure doing so will bestow a blessing upon you.{nl}I firmly believe that the Godesses have not forsaken us.
 QUEST_LV_0100_20150918_008103	
-QUEST_LV_0100_20150918_008104	We need to assemble already?{nl}Even though the world may be on the brink of destruction, I can't accept this kind of insane order.{nl}I have enough missions on my hand as it is.
-QUEST_LV_0100_20150918_008105	Alright. This should be enough.{nl}Here is a pill that instantly restores your stamina.. I'll give you one as a gift since I have enough.{nl}
+QUEST_LV_0100_20150918_008104	${memo He seems to be either heavily exaggerating, or the translation is a bit inaccurate}We need to assemble already?{nl}The world may be on the brink of destruction, but I can't accept this kind of insane order. {nl}I have enough missions on my hands as it is.
```

Comment:
Maybe you're interpreting jt wrong. I think the soldier js referring to that he cannot accept the thoughts of what happened to the world because of the Medis Diena.